### ISSUE TITLE
Compatability mode with AdGuard broken after update to AdGuard VPN v.2.3.3

### ISSUE THREAD

**Issue by AlfredSpijker** [AdguardTeam/AdGuardVPNForAndroid#210]
### Prerequisites
- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details
- **AdGuard VPN version:**
  - 2.3.3
- **VPN mode:**
  - should be Compatability mode; now SOCKS5
- **Device:**
  - OnePlus6T
- **Operating system and version:**
  - Android11
- **Root access:**
  - No

### Expected Behavior
After updating to AdGuard VPN v.2.3.3 via PlayStore (I'm a beta/alpha-tester and translator) it should work with Compatibility mode with AdGuard v.3.6.51 which is my default setting.

### Actual Behavior
Compatability mode with AdGuard is not possible, it's default set to SOCKS5. A message appears that an update of AdGuard is necessary. The upgrade button in the message leads to the default AdGuard download page. There is NO new version. 

### Additional Information
Forced closing both apps and restart them doesn't help. Also a reboot of the device doesn't. 
Both apps are running but the Always-On-VPN notice is missing.

**Comment by ameshkov** [comment]
Huh, that's a problem. There's a new code for integration with AdGuard v4.0 (which is not yet available to general public), but it should not have been pushed to GPlay.

Let us check.

**Comment by AlfredSpijker** [comment]
Thought something like that.
I don't want to run AdGuard v.4 on this device yet.
I could downgrade to AdGuard VPN v.2.2.36. Advisable?

**Comment by ameshkov** [comment]
The other solution would be switching to beta (exiting the pre-alpha access group).

**Comment by AlfredSpijker** [comment]
> The other solution would be switching to beta (exiting the pre-alpha access group).

Can I download the latest working alpha version (2.3.2?) somewhere, so I don't have to leave the alpha-group? Of course blocking updates in PlayStore untill the alpha is fixed.

**Comment by ameshkov** [comment]
Not really, GPlay does not allow downgrading

**Comment by AlfredSpijker** [comment]
OK. I removed 2.3.3 and reinstalled 2.2.36. Running fine now (of course). 
Can you inform me here if/when the next version of the alpha in PlayStore is fixed and should run with AdGuard v.3?

**Comment by ameshkov** [comment]
From this moment and on I think we won't be keeping AG v3 compatibility in the alpha channel.

It's not too long until the proper AG v4 is in the nightly channel already.

**Comment by AlfredSpijker** [comment]
> From this moment and on I think we won't be keeping AG v3 compatibility in the alpha channel.
> 
> It's not too long until the proper AG v4 is in the nightly channel already.

Ok. For your information:
For testing I updated AG (in app settings) to nightly and got v.4.0.35(79). Then I updated AG-VPN via Play Store and got v.2.3.pre-alpha1(3).
However, it's still not possible to run them in Integrated mode. In AG-VPN the SOCKS5 mode is forced. And there's still the "upgrade AG" message with a link to download AG, but that is v.3.

How can I run both nightly's in Integrated mode?
